apps/connect/source/ftrack_connect/ui/application.py

```diff
@@ -711,7 +711,13 @@ def _configure_connect_and_discover_plugins(self):
 
         self._configure_action_launcher_widget()  # Was external ftrack-connect action-launcher plugin
 
-        self._configure_plugin_manager_widget()  # Was external ftrack-connect-plugin-manager plugin
+        if (
+            os.environ.get('FTRACK_CONNECT_DISABLE_PLUGIN_MANAGER') or ''
+        ).lower() not in [
+            'true',
+            '1',
+        ]:
+            self._configure_plugin_manager_widget()  # Was external ftrack-connect-plugin-manager plugin
```

Comment:
I would do this fix on line 1199 instead, so we make sure that PluginManager class is imported in any case and if later on we will allow to install by command we will not be in need of modifying this.
Also you can use the following if that is more simple:
`if os.environ.get('FTRACK_CONNECT_DISABLE_PLUGIN_MANAGER', None):`